File: jwtpython/create.py
import jwt
import datetime

now = datetime.datetime.now()
iat = int(now.timestamp())

tomorrow = now + datetime.timedelta(days=1)
expireAt = int(tomorrow.timestamp())

user = "Alexys Lozada"
email = "alexys.lozada@gmail.com"

secret = "this-is-a-very-long-and-secure-secret"

userdata = {
        "user": user,
        "email": email,
        "iat": iat,
        "exp": expireAt,
}

token = jwt.encode(userdata, secret, algorithm="HS256")

print(token)

File: jwtgo/create.go
package main

import (
	"fmt"
	"time"

	jwt "github.com/dgrijalva/jwt-go"
)

func Create() {
	user := "Alexys Lozada"
	email := "alexys.lozada@gmail.com"
	now := time.Now()
	iat := now.Unix()
	eat := now.AddDate(0, 0, 1).Unix()
	secret := "this-is-a-very-long-and-secure-secret"

	payload := jwt.MapClaims{
		"name":  user,
		"email": email,
		"iat":   iat,
		"exp":   eat,
	}

	token := jwt.NewWithClaims(jwt.SigningMethodHS256, payload)
	tokenString, err := token.SignedString([]byte(secret))
	if err != nil {
		fmt.Println(err)
		return
	}

	fmt.Println(tokenString)
}
